ftests/nuxeo-platform-explorer-jsf-ftests/src/test/java/org/nuxeo/functionaltests/jsf/explorer/pages/ArtifactHomePage.java

```diff
@@ -38,6 +39,11 @@ public ArtifactHomePage(WebDriver driver) {
         super(driver);
     }
 
+    @Override
+    public void check() {
+        // NOOP
```

Comment:
Is it easy to find why we disable the check?

Reply:
maybe not but i'd rather not complicate JSF legacy tests. here it's just because nobody's calling this method anyway.

Reply:
Good for me 🙂